core/storage/suggestion/gae_models.py

```diff
@@ -610,3 +613,79 @@ def export_data(cls, user_id):
                 'rejection_message': voiceover_model.rejection_message
             }
         return user_data
+
+
+class CommunityContributionStatsModel(base_models.BaseModel):
+    """Records the contributor dashboard contribution stats. This includes the
+    total number of reviewers for each suggestion type and the total number of
+    suggestions in review for each suggestion type. There is only ever one
+    instance of this model, and its ID is COMMUNITY_CONTRIBUTION_STATS_MODEL_ID.
+    """
+
+    # A dictionary where the keys represent the language codes that translation
+    # suggestions are offered in and the values correspond to the total number
+    # of reviewers who have permission to review translation suggestions in
+    # that language.
+    translation_reviewer_counts_by_lang_code = ndb.JsonProperty(required=True)
+    # A dictionary where the keys represent the language codes that translation
+    # suggestions are offered in and the values correspond to the total number
+    # of translation suggestions that are currently in review in that language.
+    translation_suggestion_counts_by_lang_code = ndb.JsonProperty(required=True)
+    # The total number of reviewers who have permission to review question
+    # suggestions.
+    question_reviewer_count = ndb.IntegerProperty(required=True)
+    # The total number of question suggestions that are currently in review.
+    question_suggestion_count = ndb.IntegerProperty(required=True)
+
+    @classmethod
+    def get(cls):
+        """Gets the CommunityContributionStatsModel instance. If the
+        CommunityContributionStatsModel does not exist yet, it is created.
+        This method helps enforce that there should only ever be one instance
+        of this model.
+
+        Returns:
+            CommunityContributionStatsModel. The single model instance.
+        """
+        community_contribution_stats_model = cls.get_by_id(
+            COMMUNITY_CONTRIBUTION_STATS_MODEL_ID
+        )
+
+        if community_contribution_stats_model is None:
+            community_contribution_stats_model = cls(
+                id=COMMUNITY_CONTRIBUTION_STATS_MODEL_ID,
+                translation_reviewer_counts_by_lang_code={},
+                translation_suggestion_counts_by_lang_code={},
+                question_reviewer_count=0,
+                question_suggestion_count=0
```

Comment:
Should we have these as a default value for the properties? (exploration/gae_models for ref.)

Reply:
Changed in the follow up PR :) Thanks though - I was thinking of doing this but didn't know that you could do it at the ndb field level.

Reply:
Actually, probably better to do it here instead of within ndb.Model (for this case, at least). The reason for that is that you have better validation -- if someone else puts a model and omits a field by accident, then the required=True will catch it, whereas the default=XXX will just silently set the missing value to the default.

Reply:
Okay - thanks Sean! I'll make the change.